modules/ob-role-based-access-control.adoc

```diff
@@ -0,0 +1,14 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: CONCEPT
+[id="ob-role-based-access-control_{context}"]
+= Role-based Access Control
```

Comment:
Does this need to be init capped all the way or is it just a general reference to RBAC?
```suggestion
= Role-based access control
```